apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.sql

```diff
@@ -0,0 +1,55 @@
+-- BLOCK select_course_instance
```

Comment:
Let's update the name to differentiate this from normal `course_instances`.

```suggestion
-- BLOCK select_lti13_course_instance
```